JAMES-3302 Keep improving Distributed Server documentation (#237)
apache/james-project: docs/modules/servers/pages/distributed/operate/webadmin.adoc

* 200: List of available health checks

== Task management

> Some webadmin features schedules tasks. The task management API allow to

Reviewer: ```suggestion
Some webadmin features schedule tasks. The task management API allow to
```

....

An Execution Report will be returned:

....
{
    "submitDate": "2017-12-27T15:15:24.805+0700",
    "startedDate": "2017-12-27T15:15:24.809+0700",
    "completedDate": "2017-12-27T15:15:24.815+0700",
    "cancelledDate": null,
    "failedDate": null,
    "taskId": "3294a976-ce63-491e-bd52-1b6f465ed7a2",
    "additionalInformation": {},
    "status": "completed",
    "type": "type-of-the-task"
}
....

Note that:

* `status` can have the value:
** `waiting`: The task is scheduled but its execution did not start yet
** `inProgress`: The task is currently executed
** `cancelled`: The task had been cancelled
** `completed`: The task execution is finished, and this execution is a
success
** `failed`: The task execution is finished, and this execution is a
failure
* `additionalInformation` is a task specific object giving additional
information and context about that task. The structure of this
`additionalInformation` field is provided along the specific task
submission endpoint.

Response codes:

* 200: The specific task was found and the execution report exposed
above is returned
* 400: Invalid task ID
* 404: Task ID was not found

=== Awaiting a task

> One can await the end of a task, then receive it’s final execution

Reviewer: ```suggestion
One can await the end of a task, then receive its final execution
```
